cmd/prober/write.go

```diff
@@ -121,17 +122,33 @@ func rekorWriteEndpoint(ctx context.Context) error {
 	t := time.Now()
 	resp, err := http.DefaultClient.Do(req)
 	latency := time.Since(t).Milliseconds()
+	exportDataToPrometheus(resp, rekorURL, endpoint, POST, latency)
 	if err != nil {
-		fmt.Printf("error adding entry: %v\n", err.Error())
+		return fmt.Errorf("error adding entry: %w", err)
 	}
 
-	// Export data to prometheus
-	exportDataToPrometheus(resp, rekorURL, endpoint, POST, latency)
-
+	// If entry was added successfully, we should verify it
+	rekorClient, err := rclient.GetRekorClient(rekorURL, rclient.WithUserAgent(fmt.Sprintf("Sigstore_Scaffolding_Prober/%s", versionInfo.GitVersion)))
+	if err != nil {
+		return fmt.Errorf("creating rekor client: %w", err)
+	}
 	defer resp.Body.Close()
-	body, _ = io.ReadAll(resp.Body)
-	fmt.Println(string(body))
-	return nil
+	var logEntry models.LogEntry
+	err = json.NewDecoder(resp.Body).Decode(&logEntry)
+	if err != nil {
+		return fmt.Errorf("unmarshal: %w", err)
+	}
```

Comment:
Should we also test that we can re-fetch this entry from the log instead of just using the response from earlier? If so, should that be a new metric?

Reply:
we already test that we can fetch an entry from the log, so we should be good on that. with the current setup we can also be confident that if the verification succeeds that the entry exists, so we should be good!